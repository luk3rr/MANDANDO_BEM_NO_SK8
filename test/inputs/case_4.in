10 4
10 67
8 50
10 87
10 74
10 59
9 38
4 4
8 39
5 20
8 70
1 5
8 7
12 11
17 19
