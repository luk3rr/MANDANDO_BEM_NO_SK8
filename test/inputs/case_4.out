7150
4 4 3 2 1
0
4 4 3 2 1
0
4 4 3 2 1
3 4 3 2
0
3 4 3 2
0
4 4 3 2 1
